.github/workflows/testing.yml

```diff
@@ -93,13 +88,13 @@ jobs:
         if: matrix.buildsys == 'make'
         run: |
           export PATH="/usr/local/opt/bison/bin:$PATH"
-          make develop -j Q= CXXFLAGS="-O3 -flto -DNDEBUG -mmacosx-version-min=10.9"
+          CXXFLAGS="-ggdb3 -Og -fno-omit-frame-pointer -fno-optimize-sibling-calls -mmacosx-version-min=10.9 -I include/hacks" make develop -ej Q=
```

Comment:
Don't we typically do `make VARS="values"` not `VARS=values" make`?

Reply:
Yes, but they work and serve different purposes.
- `make VAR=value` passes `VAR` as a Make variable.
- `VAR=value make` passes `VAR` as an **environment** variable.

The difference is that `make develop` overrides the Make variables `WARNFLAGS` and `CXXFLAGS`, which means that `make develop CXXFLAGS="never used"` does... well, you see.

The strategy here is to pass it as an environment variable (which are inherited by children processes, including the sub-`make`), and tell Make that environment variables override internal ones (`make -e`).

Yeah, that's a hack. But it should be functional.Yes, but they work and serve different purposes.
- `make VAR=value` passes `VAR` as a Make variable.
- `VAR=value make` passes `VAR` as an **environment** variable.

The difference is that `make develop` overrides the Make variables `WARNFLAGS` and `CXXFLAGS`, which means that `make develop CXXFLAGS="never used"` does... well, you see.

The strategy here is to pass it as an environment variable (which are inherited by children processes, including the sub-`make`), and tell Make that environment variables override internal ones (`make -e`).

Yeah, that's a hack. But it should be functional.